apps/pom.xml

```diff
@@ -297,9 +299,17 @@
 
 			<dependency>
 				<!-- The test framework used for unit and integration tests. -->
-				<groupId>junit</groupId>
-				<artifactId>junit</artifactId>
-				<version>4.13.1</version>
+				<groupId>org.junit.jupiter</groupId>
+				<artifactId>junit-jupiter-engine</artifactId>
+				<version>${junit.version}</version>
+				<scope>test</scope>
+			</dependency>
+
+			<dependency>
+				<!-- Allows junit compatibility with old junit4 tests -->
+				<groupId>org.junit.vintage</groupId>
+				<artifactId>junit-vintage-engine</artifactId>
+				<version>${junit.vintage.version}</version>
```

Comment:
I wonder whether we should just make this a dependency in this pom so that we don't need it in every single other pom. Not something that I would suggest for practically any other dependency but this is ubiquitous. Do we have any projects that don't need junit as a test dependency?

Reply:
There are some submodules that didn't have junit listed as a dependency, at least.